### PR TITLE
website: Add FAQ section

### PR DESCRIPTION
**Pull request by vyncent-t**
# Update Website to Reflect New FAQ Section in Documentation

Related to PR #1547 headlamp repo

## Description
This PR updates the Headlamp website to reflect the addition of the new FAQ section in the documentation. It ensures that visitors to the website have easy access to the FAQ section, thereby enhancing user experience and providing quick access to common questions and answers.

## Changes
- [x] Updated website navigation to include a link to the new FAQ section.
- [x] Ensured consistency in the website’s footer with the addition of the FAQ link, mirroring the changes in the documentation repo.

## Verification
- [x] Tested all new links on the website to ensure they correctly redirect to the FAQ section in the documentation.
- [x] Coordinated with documentation updates to ensure consistency across platforms.

## Screenshots 
image

### CODE REVIEW

File: config.yaml [hunk 1]
```diff
@@ -45,8 +45,10 @@ params:
             link: https://headlamp.dev/docs/latest/installation/desktop/
           - title: Plugin development
             link: https://headlamp.dev/docs/latest/development/plugins/
+          - title: Frequently Asked Questions
```
Comment (joaquimrocha) [inline config.yaml:48]: I'd prefer `F.A.Q.`. I think it more quickly catches the eye.
Up to you @blixtra

**Comment by blixtra** [comment]
Apart from @joaquimrocha's suggestion, LGTM.

File: config.yaml (hunk 1 above)
Comment (vyncent-t) [reply config.yaml:48]: done!

File: config.yaml [hunk 2]
```diff
@@ -45,8 +45,10 @@ params:
...
+            link: https://headlamp.dev/docs/latest/faq/
   news_stripe:
-    text: 'Headlamp present with a kiosk at KubeCon NA 2023 [Read more »](/blog/2023/11/headlamp-in-kubecon-north-america-2023/)'
+    text: "Headlamp present with a kiosk at KubeCon NA 2023 [Read more »](/blog/2023/11/headlamp-in-kubecon-north-america-2023/)"
```
Comment (vyncent-t) [inline config.yaml:51]: set to single

**Review by joaquimrocha — APPROVED** [review]

**Pull request merged**